### PR TITLE
Implement New JobPosting XML API

### PR DESCRIPTION
**Pull request by lukasbischof**

### CODE REVIEW

File: app/views/home/index.de.html.slim [hunk 1]
```diff
@@ -21,14 +21,14 @@ section.jobs
     .row
       - JobPosting.order(publication_date: :desc).first(6).each do |posting|
         .col-12.col-lg-6.mb-4
-          = link_to posting.link, target: '_blank' do
+          = link_to '#', target: '_blank' do
```
Comment (lukasbischof) [inline app/views/home/index.de.html.slim:24]: temporary until I implemented a show

File: app/jobs/job_posting_poll_job.rb [hunk 2]
```diff
@@ -4,6 +4,6 @@ class JobPostingPollJob < ApplicationJob
   queue_as :default
 
   def perform(*)
-    JobPostingPollerService.poll
+    JobPostingApi::Poller.new.perform
```
Comment (TheFehr) [inline app/jobs/job_posting_poll_job.rb:7]: Don't you wan't perform_later? And then sidekiq starts as soon as it's ready? Maybe in the future we have many more ActiveJobs?

Comment (lukasbischof) [reply app/jobs/job_posting_poll_job.rb:7]: It's calling the service. The job itself you still can perform_later

Comment (TheFehr) [reply app/jobs/job_posting_poll_job.rb:7]: Aaah, you named the "execute" function of your Service perform as well. That confused me.

**Review by TheFehr — APPROVED** [review]

**Pull request merged**